py/test_crypto_concurrent.py

```diff
@@ -20,7 +20,11 @@ async def test_crypto_test_app(n):
 
         # In real situations we may have gotten control over several endpoints,
         # but for us we know there are no grouped endpoints
-        endpoint = endpoints[0]
```

Comment:
Did you encounter this?
If so I'm guessing you ran the server without a config file which had an endpoint with a "mocks" label?

I agree on your change here.
But the server will actually send an error response to the python client when this situation happens.
I intend to raise an exception in the python client if that happens, just haven't gotten around to it.